### DISCUSSION TITLE
Any exercises to practice phase field to faciliate understanding?

### DISCUSSION THREAD

**Discussion by ZhigangPu** [idaholab/moose#19342] [Q&A Modules: Phase field] [answered]
Dear,

(Feel free to point out if it is not appropriate to pose such questions here :))

I wonder is there some book about phase field that has exercises? I've read books <Phase-field methods in Material Science and Engineering> written by Nikolas Provatas and I also downloaded some sample code to play with but I still feel that I don't really understand phase field, I think I may need some exercises to help me understand.

Best,
zhigang

**Comment by GiudGiud** [comment, marked as answer]
There s a tutorial here
https://mooseframework.inl.gov/modules/phase_field/Tutorial.html

**Reply by GiudGiud** [reply]
and some good info in general on this page
https://mooseframework.inl.gov/modules/phase_field/index.html

**Comment by ZhigangPu** [comment]
Thanks a lot, I'll read through it!
